APMrover2/fence.cpp

```diff
@@ -16,13 +16,36 @@ void Rover::fence_check()
 
     // if there is a new breach take action
     if (new_breaches) {
+    	gcs().send_text(MAV_SEVERITY_NOTICE,"Geofence triggered");
```

Comment:
I'm not a huge fan of send_texts if we can avoid them and I think the MP at least displays a warning to the user based on the https://mavlink.io/en/messages/common.html#FENCE_STATUS mavlink message.